app/src/userpages/i18n/en.po

```diff
@@ -420,26 +432,48 @@ msgstr "Partitions"
 msgid "userpages.streams.edit.details.nav.streamPartitions"
 msgstr "Stream partitions"
 
-msgid "userpages.streams.edit.details.name"
-msgstr "Name"
+msgid "userpages.streams.edit.details.info.description"
+msgstr ""
+"All streams require a unique path in the format <strong>domain/pathname</strong>. Choose a unique name, "
+"and create your stream in the default <strong>%{defaultDomain}</strong> domain or "
```

Comment:
Should we stick to "Stream" here as well (Stream vs stream)? Do we have a rule about the way we capitalize such words?

Reply:
Good question, one for @mattatgit